### PR TITLE
enhance: log kafka event mobile

### PR DESCRIPTION
**Pull request by indraprasetya154**
## Overview
- enhance: log kafka event mobile
- log event will be same with website

cc: @jabardigitalservice/jds-backend 

## Related PR
https://gitlab.com/jdsteam/sikd-website/-/merge_requests/493

## Evidence
title: enhance: log kafka event mobile
project: SIKD
participants: @indraprasetya154 @samudra-ajri @fajarhikmal214 @azophy

### CODE REVIEW

**Review by fajarhikmal214 — APPROVED** [review]

**Review by samudra-ajri — APPROVED** [review]
Awesome!
However, I think we sould decide the standard attribute for the log. For instance, we could define the rule to write the event and its status. 😁

File: src/app/Http/Traits/SignatureTrait.php [hunk 1]
```diff
@@ -67,17 +68,19 @@ public function invalidResponseCheckUserSignature($checkUserResponse)
...
         ];
 
         if ($checkUserResponse->status_code == BsreStatusTypeEnum::RESPONSE_CODE_BSRE_ACCOUNT_NOT_REGISTERED()->value) {
+            $logData['event'] = 'esign_check_user_status_not_registered';
             $logData['message'] = 'Invalid User NIK';
             $logData['longMessage'] = 'User NIK Anda Belum Terdaftar';
         }
 
         if ($checkUserResponse->status_code == BsreStatusTypeEnum::RESPONSE_CODE_BSRE_ACCOUNT_ESIGN_NOT_ACTIVE()->value) {
+            $logData['event'] = 'esign_check_user_status_not_have_certified';
```
Comment (samudra-ajri) [inline src/app/Http/Traits/SignatureTrait.php:83]: In my opinion, these events could have the same event, e.g. esign_check_user_status. Then, each of them shoud has status based on the response, for examples:
- status: FAILED_INVALID_BSRE
- status: FAILED_NOT_REGISTERED
- status: FAILED_NOT_CERTIFIED

This is more conveniet to build the filter for the comparison chart.

Comment (indraprasetya154) [reply src/app/Http/Traits/SignatureTrait.php:83]: oh i see, event should be main log and status should be detail of event, right? @samudra-ajri

Comment (samudra-ajri) [reply src/app/Http/Traits/SignatureTrait.php:83]: Correct, Sir.. 😁

Comment (indraprasetya154) [reply src/app/Http/Traits/SignatureTrait.php]: kindly check again dude

**Review by azophy — APPROVED** [review]

File: src/app/Http/Traits/SignatureTrait.php (hunk 1 above)
Comment (samudra-ajri) [reply src/app/Http/Traits/SignatureTrait.php]: save.. 😁

**Pull request merged**